<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qsetmaster</class>
 <widget class="QFrame" name="qsetmaster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Master</string>
  </property>
  <property name="toolTip">
   <string>Set Master allows creation, viewing, deletion, and moving of sets.</string>
  </property>
  <property name="whatsThis">
   <string>The Set Master</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>778</width>
     <height>412</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="setGridLayout"/>
    </item>
    <item row="0" column="1">
     <widget class="QTableWidget" name="m_set_table">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>248</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>340</height>
       </size>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>32</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>21</number>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="m_set_contents_text">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>156</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>156</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QPushButton" name="m_button_show">
        <property name="toolTip">
         <string>Show a condensed list of sets and their patterns.</string>
        </property>
        <property name="text">
         <string>Show Sets Tracks</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="m_button_down">
        <property name="toolTip">
         <string>Move the selected set down in the list.</string>
        </property>
        <property name="text">
         <string>Set Down</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="m_button_up">
        <property name="toolTip">
         <string>Move the selected set up in the list.</string>
        </property>
        <property name="text">
         <string>Set Up</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="m_set_number_text"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="m_set_name_text">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="m_button_delete">
        <property name="toolTip">
         <string>Clears the selected set. No undo! Careful!</string>
        </property>
        <property name="text">
         <string>Clear Selected Set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Set No.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_set_table</tabstop>
  <tabstop>m_set_contents_text</tabstop>
  <tabstop>m_set_number_text</tabstop>
  <tabstop>m_set_name_text</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>m_button_down</tabstop>
  <tabstop>m_button_up</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>m_button_show</tabstop>
  <tabstop>m_button_delete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
